src/ai/.x/templates/openai-chat-js/Main.js

```diff
@@ -1,21 +1,90 @@
-const { OpenAI } = require('openai');
+{{if {_IS_AUTHOR_COMMENT} }}
+ // Showcasing syntax differences for the same content:
+ //   The Learn docs use CommonJS module syntax, which includes `require` and `module.exports`
+ //     This is an older standard for working with modules in JavaScript and is widely supported, especially in Node.js.
+ //   The Studio "view code" uses ES6 module syntax, which includes `import` and `export`.
+ //     This is a newer standard for working with modules in JavaScript and is supported in modern browsers and Node.js versions
+{{endif}}
+{{if {_IS_STUDIO_VIEW_CODE_TEMPLATE}}}
+import { AzureOpenAI } from "openai";
+import { DefaultAzureCredential, getBearerTokenProvider } from "@azure/identity";
+{{else if {_IS_LEARN_DOC_TEMPLATE} || !{_IS_STUDIO_VIEW_CODE_TEMPLATE}}}
+const { OpenAI } = require("openai");
+{{endif}}
+{{if !{_IS_LEARN_DOC_TEMPLATE} || !{_IS_STUDIO_VIEW_CODE_TEMPLATE}}}
 const { {ClassName} } = require("./OpenAIChatCompletionsClass");
 const { readline } = require("./ReadLineWrapper");
+{{endif}}
 
+{{if {_IS_LEARN_DOC_TEMPLATE}}}
+// Load the .env file if it exists
```

Comment:
I see what I did.... need to move these (and instances below) to the ./includes/*.js file